src/Solver/time_stepping/AbstractGhostTimeCluster.h

```diff
@@ -43,5 +42,7 @@ class AbstractGhostTimeCluster : public AbstractTimeCluster {
   void reset() override;
   ActResult act() override;
 
+  ClusterTimes& getClusterTimes();
+  std::vector<seissol::time_stepping::NeighborCluster>* getNeighborClusters();
```

Comment:
If we want to forward them, then it's maybe better to do that in the `AbstractTimeCluster` already

Reply:
Done